versionutils/repo.go

```diff
@@ -83,6 +105,7 @@ func ParseTomlFromDir(relativeDir string) ([]*toml.Tree, error) {
 	return parseTomlFromDir(relativeDir, constraint)
```

Comment:
let's deprecate this function and add ParseTomlConstraintsFromDir

Reply:
oops 👍